authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from authors.apps.profiles.views import (UserProfileView, UpdateUserProfileView, UserListView)
+
+app_name = 'profiles'
+
+urlpatterns = [
+    path('profiles/<str:username>', UserProfileView.as_view(), name='profile'),
+    path('profiles/<str:username>/', UpdateUserProfileView.as_view(), name='update_profile'),
+    path('profiles/', UserListView.as_view(), name='list_users'),
```

Comment:
Hello @Philipotieno ,
Add tests for this endpoint. It is currently untested

Reply:
Just added the tests @Ogutu-Brian